sunpy/visualization/imageanimator.py

```diff
@@ -1,19 +1,16 @@
 # -*- coding: utf-8 -*-
 
 import abc
-from sunpy.extern import six
 
 import numpy as np
 
 import matplotlib.pyplot as plt
 import matplotlib.widgets as widgets
 import matplotlib.animation as mplanim
-
-import astropy.units as u
```

Comment:
I think you still need to import astropy.units.  My previous comment was just to delete the blank line above and below.